tests/console/upgrade_snapshots.pm

```diff
@@ -23,12 +23,16 @@ sub run {
     select_console 'root-console';
 
     my $waittime = 200 * get_var('TIMEOUT_SCALE', 1);
-    script_run("snapper list | tee /dev/$serialdev", 0);
+    my $ret      = script_run("snapper --help | grep disable-used-space");
+    my $disable  = '';
```

Comment:
It is better to make it in one line, such as:
my $disable  = $ret ? '' : '--disable-used-space';

Reply:
ok